static/docker-compose.yml

```diff
@@ -169,6 +170,41 @@ services:
     entrypoint: npm run
     command: "start:notification-sender"
 
+  datalake-maintainer:
+    image: "plextrac/plextracapi:${UPGRADE_STRATEGY:-stable}"
+    deploy:
+      replicas: 0
+    depends_on:
+    - postgres
+    - plextracdb
+    - redis
+    restart: always
+    environment:
+      CB_API_PASS: "${CB_API_PASS:?err}"
+      CB_API_USER: "${CB_API_USER:?err}"
+      CLIENT_DOMAIN_NAME: ${CLIENT_DOMAIN_NAME:?err}
+      COUCHBASE_URL: "http://plextracdb"
+      GCP_DATALAKE_SA_KEY_PATH: "${GCP_DATALAKE_SA_KEY_PATH:-/usr/src/plextrac-api/keys/gcp/gcp-datalake-sa-key.json}"
+      LOG_LEVEL: ${LOG_LEVEL:-info}
+      REDIS_CONNECTION_STRING: "${REDIS_CONNECTION_STRING:-redis}"
+      REDIS_PASSWORD: "${REDIS_PASSWORD:?err}"
+      PG_CORE_DB: ${PG_CORE_DB:?err}
+      PG_CORE_RO_PASSWORD: ${PG_CORE_RO_PASSWORD:?err}
+      PG_CORE_RO_USER: ${PG_CORE_RO_USER:?err}
+      PG_CORE_RW_PASSWORD: ${PG_CORE_RW_PASSWORD:?err}
+      PG_CORE_RW_USER: ${PG_CORE_RW_USER:?err}
```

Comment:
Hey @evanhsu in chatting with Ben he's only doing reads against Postgres (via the service layer). Has the `RO` user been hooked up to service those requests or is it still using `RW`?

Reply:
As of right now, only the RW user is used to establish a connection to Postgres:
https://github.com/PlexTrac/product-core-backend/blob/main/src/db/core/index.ts#L17

[I'm tracking this conversation in Slack](https://plextrac.slack.com/archives/C02BSEY93T5/p1668024770169389)
and we have this Jira story that could be pulled in if necessary: https://plextrac.atlassian.net/browse/QUICK-675

Reply:
For now I'll leave this one as-is (eg, the `RW` user will be provided) but that can be rolled back when the `RO` user is pushed in. AFAIK the `datalake-maintainer` is just accessing service methods so updating those to use the `RO` account will do the trick.

This is definitely something that must happen soon before we take on more tech debt with the `RW` account, but not an emergency.